components/button/index.module.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/variables.scss';
+
+.primary {
+	padding: 6px 12px;
+	font-size: $font-size-medium-body;
+	font-weight: bold;
```

Comment:
Please add the `weight` variable like the `Typography` component.

Reply:
will we have more than one font-weight for the button?

Reply:
Maybe not, but it's good practice :)
Also, we need a transparent version of the button as well.

Reply:
Added weight variable as suggested 👍